docs/src/main/sphinx/admin/properties-optimizer.rst

```diff
@@ -187,3 +187,15 @@ Enables approximation of the output row count of filters whose costs cannot be
 accurately estimated even with complete statistics. This allows the optimizer to
 produce more efficient plans in the presence of filters which were previously
 not estimated.
+
+``optimizer.join-partitioned-build-min-row-count``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``1000000``
+* **Min allowed value:** ``0``
+
+A minimum number of join build side rows required to use partitioned join lookup.
+If a join build side number of rows is estimated, with high confidence, to be smaller,
```

Comment:
If the build side of a join is estimated to be smaller than the configured threshold, single threaded join lookup is used to improve join performance.